quiz/QuizData.scala

```diff
@@ -70,8 +70,8 @@ object QuizData {  // to generate tables for a concept connection quizes in late
       "kompilerad     " -> "maskinkod sparad och kan köras igen utan kompilering",
       "skript         " -> "maskinkod sparas ej utan skapas vid varje körning",
       "objekt         " -> "samlar variabler och funktioner",
-      "main           " -> "där exekveringen av kompilerat program startar",
-      "programargument" -> "överförs via parametern args i main",
+      "@main          " -> "där exekveringen av kompilerat program startar",
+      "programargument" -> "kan överföras via parametern args i en primitiv main",
```

Comment:
You can also give params to an @main functions. So remove "primitiv" and perhaps better with "till" instead of "i"